src/Controller/HearingController.php

```diff
@@ -136,7 +136,7 @@ public function hearingPostCreate(CaseEntity $case, DocumentUploader $documentUp
             $fileName = $mailTemplateHelper->renderMailTemplate($hearingPost->getTemplate(), $case);
 
             // Compute fitting name
-            $updatedFileName = $slugger->slug($hearingPost->getTemplate()->getName()).'-'.uniqid().'.pdf';
+//            $updatedFileName = $slugger->slug($hearingPost->getTemplate()->getName()).'-'.uniqid().'.pdf';
```

Comment:
Delete this line (and the comment above).

Reply:
Deleted